build.sbt

```diff
@@ -1,4 +1,5 @@
-import VersionHelper.{versionFmt, fallbackVersion}
+import VersionHelper.fallbackVersion
+import VersionHelper.versionFmt
```

Comment:
Sorry for formatting noise. I'm an Emacs user and I have it configured to auto-format via scalafmt on save. I was too lazy to turn it off for this PR, which unfortunately caused some noise.